src/main/java/org/ihtsdo/buildcloud/config/Config.java

```diff
@@ -106,11 +103,9 @@ public HibernateTransactionManager transactionManager(
 	}
 
 	@Bean
-	public S3ClientFactory s3ClientFactory(@Value("${aws.key}") final String accessKey,
-	                                       @Value("${aws.privateKey}") final String privateKey,
-	                                       @Value("${srs.build.s3.offline.directory}") final String directory) throws IOException {
+	public S3ClientFactory s3ClientFactory(@Value("${srs.build.s3.offline.directory}") final String directory) throws IOException {
 		s3ClientFactory = new S3ClientFactory();
-		s3ClientFactory.setOnlineImplementation(new S3ClientImpl(new BasicAWSCredentials(accessKey, privateKey)));
+		s3ClientFactory.setOnlineImplementation(new S3ClientImpl(software.amazon.awssdk.services.s3.S3Client.builder().credentialsProvider(ProfileCredentialsProvider.create()).build()));
```

Comment:
@QuyenLy87 Can we use default region builder instead please?

Reply:
The code has been updated. Thanks @CoderMChu